addons/campsite/helper/fn_createMarkerArea.sqf

```diff
@@ -0,0 +1,52 @@
+#include "script_component.hpp"
```

Comment:
rename this function to have it only be used for campsite like `createCampsiteMarkerArea`. Just to avoid others or me to use it for any more common use-cases without knowing there is a "_campsite" string in the markername